fda-model/alternatives/Alternative.cs

```diff
@@ -28,7 +28,7 @@ public Scenario FutureYearScenario
             get { return _futureYear; }
         }
 
-
+        [Obsolete("This object is obsolete. There is no need to create an alternative. We only require alternative results.")]
```

Comment:
Why are we leaving this method in? For unit tests? If we can get rid of it, lets get rid of it.

Reply:
done